scripts/build-js

```diff
@@ -0,0 +1,8 @@
+#! /usr/bin/env bash
+
+## Move the global navigaiton module
```

Comment:
that doesn't move it, and it's 'navigation'

Also seems a completely unnecessary comment, we can all see the `cp` but doesn't tell us _why_ it's copied (or indeed, if it's a mistake for it to be a `cp` where it should be a `mv`)

Reply:
Good point, updated

Reply:
Yes I wasn't sure if this comment is really necessary. I'm a fan of comments, and usually push back against those who find them obnoxious (because not everyone reading the code is as well-versed in it as you are), but this one does seem a little redundant. @anthonydillon do you think we have a single dev who might not inherently understand a `cp` command?

Reply:
Sure, comments removed.